app/scrap_paper.py

```diff
@@ -0,0 +1,40 @@
+quick_reply_use_of_force = [
+    {
+        "label": "Rank 1",
+        "description": "Non-violent Police Presence.",
+        "metadata": "force_rank_1"
+    },
+    {
+        "label": "Rank 2",
+        "description": "Open Handed (Arm Holds & Pushing)",
+        "metadata": "force_rank_2"
+    },
+    {
+        "label": "Rank 3",
+        "description": "Blunt Force Trauma (Batons & Shields)",
+        "metadata": "force_rank_3"
+    },
+    {
+        "label": "Rank 4",
+        "description": "Chemical & Electric Weapons (Tasers & Pepper Spray)",
+        "metadata": "force_rank_4"
+    },
+    {
+        "label": "Rank 5",
+        "description": "Lethal Force (Guns & Explosives)",
+        "metadata": "force_rank_5"
+    }
+]
+
+confirmation_reply = [
+    {
+        'label': 'Yes',
+        'description': 'Yes I can provide more information',
+        'metadata': 'confirm_yes'
+    },
+    {
+        'label': 'No',
+        'description': 'No I can\'t provide more information',
```

Comment:
the '/' in can't should be removed.  So far it's looking good though!

Reply:
@Andrew-Haney without the \ wouldn't compiler think the code ends after can? or did you mean have it as cant instead of can't?

Reply:
Yeah, so the \ escapes the ' otherwise the string would end at can. I use single quotes for almost everything and this is one of the reasons not to do it.